.github/workflows/api-tests.yml

```diff
@@ -0,0 +1,37 @@
+name: API-Tests
+on:
+  pull_request:
+    types: [ready_for_review]
+
+jobs:
+  api-tests:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      # chagne this to use docker compose rather than docker-compose 
+      # this step is only required locally when using act since the ubuntu image
+      # does not come with docker-compose installed, however the alternative to the reinstall
+      # here is you'd have to install a custom docker image locally which includes docker-compose
+      - name: Install docker-compose
+        run: | 
+          sudo curl -L https://github.com/docker/compose/releases/download/1.29.2/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
+          sudo chmod +x /usr/local/bin/docker-compose
+
+      - name: Run docker-compose
+        working-directory: ./api
+        run: DATACITE_USER=${{ secrets.DATACITE_USER }} DATACITE_PASSWORD=${{ secrets.DATACITE_PASSWORD }} ORCID_ID=${{ secrets.ORCID_ID }} ORCID_SECRET=${{ secrets.ORCID_SECRET }} docker-compose -f "docker-compose.yml" up -d --build
```

Comment:
github recommend these are set in an 'env:' block not on the command line (though it does mask them it may sometimes  fail to), env blocks are not logged

Reply:
fixed